### PR TITLE
Add itosho/easy-query

### PR DESCRIPTION
**Pull request by itosho**
Please check this. 🙏 
- https://packagist.org/packages/itosho/easy-query

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -313,6 +313,7 @@ Additional lists you might find useful:
 - [Serializeable Data Types plugin](https://github.com/burzum/cakephp-serialize-data-types) - Serialize DB content as JSON or using phps serializing functions.
 - :strawberry: [Muffin/Webservices ORM plugin](https://github.com/usemuffin/webservice) - An ORM like interface for webservices.
 - :strawberry: [Connehito/CakephpMasterReplica plugin](https://github.com/Connehito/cakephp-master-replica) - Switch master/replica database connections.
+- [itosho/easy-query plugin](https://github.com/itosho/easy-query) - Behavior for easily some complicated queries.
```
Comment (ADmad) [inline README.md:316]:
```suggestion
- [itosho/easy-query plugin](https://github.com/itosho/easy-query) - Behavior for easily generating some complicated queries like (bulk) insert/upsert etc.
```

**Review by ADmad — COMMENTED** [review]
Nice, seems useful.

File: README.md (hunk 1 above)
Comment (dereuromark) [reply README.md:316]: The plugin name is also invalid here.

**Comment by dereuromark** [comment]
"Behavior for easily some complicated queries"
I have seen this sentence also in the plugin a few times - but there is a word missing for it.

**Comment by itosho** [comment]
Thank you for the suggestion.
(My English is not very good.)

File: README.md [hunk 2]
```diff
@@ -313,6 +313,7 @@ Additional lists you might find useful:
 - [Serializeable Data Types plugin](https://github.com/burzum/cakephp-serialize-data-types) - Serialize DB content as JSON or using phps serializing functions.
 - :strawberry: [Muffin/Webservices ORM plugin](https://github.com/usemuffin/webservice) - An ORM like interface for webservices.
 - :strawberry: [Connehito/CakephpMasterReplica plugin](https://github.com/Connehito/cakephp-master-replica) - Switch master/replica database connections.
+- [itosho/easy-query plugin](https://github.com/itosho/easy-query) - Behavior for easily generating some complicated queries like (bulk) insert/upsert etc.
```
Comment (dereuromark) [inline README.md:316]:
```suggestion
- [Itosho/EasyQuery plugin](https://github.com/itosho/easy-query) - Behavior for easily generating some complicated queries like (bulk) insert/upsert etc.
```

**Comment by dereuromark** [comment]
The `>=7.1` requirement for PHP seems to be missing in the composer file
That would lead to problems if people can install it but it will break due to typehints.

**Comment by itosho** [comment]
Thanks. I fix it soon.

**Comment by itosho** [comment]
@dereuromark I added the requirement for PHP. Please check again 🙏 
ref: https://github.com/itosho/easy-query/pull/31

**Pull request merged**